project_template/models/project_task.py

```diff
@@ -0,0 +1,53 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+from odoo.osv.expression import AND
+
+
+def should_apply_default_template_filter(domain, context):
+    template_field_in_domain = any(
+        isinstance(el, (list, tuple)) and el[0] == 'is_template'
+        for el in domain
+    )
+    return not (template_field_in_domain or context.get('show_task_templates'))
+
+
+class ProjectTask(models.Model):
+
+    _inherit = 'project.task'
+
+    is_template = fields.Boolean()
+
+    @api.model
+    def _get_values_for_invisible_template_fields(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4397703)